spec/requests/customer_resources_spec.rb

```diff
@@ -72,6 +72,11 @@
       it 'has reason ' do
         expect(attributes.visibilityData).to have_key(:reason)
       end
+      it 'has reason empty' do
+        expect(attributes.visibilityData.reason).to(
+          eq('')
+        )
```

Comment:
while this is valid, it'd be slightly more idiomatic ruby to drop the parens from `to`, like so:
```
expect(attributes.visibilityData.reason).to eq('')
```

Reply:
will take care of this, thanks